CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 
 ## 0.23.0
 
+* Make it possible to configure maximum number of connections maximum connection creation rate in listener configuration
```

Comment:
maybe it's missing an "and"? ..." maximum number of connections AND maximum connection creation rate"